docs/spectral_cube.rst

```diff
@@ -85,7 +85,7 @@ spectral regions from the cube.
 
     >>> import astropy.units as u
     >>> from specutils.manipulation import spectral_slab
-    >>> ss = spectral_slab(sc, 5000.*u.AA, 5003.*u.AA)
+    >>> ss = spectral_slab(sc, 5000.*u.AA, 5003.*u.AA) # doctest: +REMOTE_DATA
```

Comment:
xref #835

Reply:
That's in fact an interesting point – since loading `sc` has already been decorated `+REMOTE_DATA` above, what would a non-remote test actually do here if the spectrum is not available?

Reply:
Any code that uses the remote data product is basically remote data and needs to be marked as such to be tested properly. If you don't add it here, you'll get exception.

Reply:
@dhomeier It would error because of `sc` being undefined (as I saw earlier...)